### ISSUE TITLE
Logic error when two pool groups defined

### ISSUE THREAD

**Issue by Schnitzel** [UpstreamData/pyasic#54]
this:
https://github.com/UpstreamData/pyasic/blob/b5d2809e9ce05df7e169ddd183caf055f4099f4f/pyasic/miners/base.py#L461-L462

should probably this way:

```
miner_data["pool_2_url"] = pools_data[1]["pool_1_url"]
miner_data["pool_2_user"] = pools_data[1]["pool_1_user"]
```

**Comment by UpstreamData** [comment]
Yeah it appears you're correct.  I'll get that fixed here quickly.

**Comment by UpstreamData** [comment]
Fixed in 0.36.6
